opencensus/stats/measure_to_view_map.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.tags import tag_map
+from opencensus.stats.measurement import Measurement
+from opencensus.stats import measurement
+from opencensus.stats import view
+from opencensus.stats import view_data
+from datetime import datetime
+import threading
+
+
+def synchronized(lock):
+
+    def wrap(method):
+        def new_method(self, *args, **kwargs):
+            yield from lock
```

Comment:
`yield from` is introduced in Python3.3+, please make your code python2&3 compatible.

Reply:
https://stackoverflow.com/questions/17581332/converting-yield-from-statement-to-python-2-7-code